host/3.0/bionic/arm32v7/dotnet/dotnet.Dockerfile

```diff
@@ -22,19 +22,19 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
     unzip /$EXTENSION_BUNDLE_FILENAME -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
     rm -f /$EXTENSION_BUNDLE_FILENAME && \
-    EXTENSION_BUNDLE_VERSION_V2=2.21.0 && \
```

Comment:
You will notice all of these changes are under the "host/3.0" folder. Since we are not releasing v3 anymore, we should not make changes to this folder. Instead, please change the dockerfiles in the "host/4" folder.